src/helpers/TimeHelper.ts

```diff
@@ -17,6 +17,10 @@ export function parseTimeRange(input: string): ITimeRange | string {
         return durationResult;
     }
 
+    if (15 % durationResult == 0) {
+        return `${time} is not rounded to the nearest quarter`;
```

Comment:
I would argue a warning is better, and then just only show two decimals. Approving nonetheless as that might be gold plating.

Reply:
I agree. I thought I'd implement it in #16 and leave the functionality for now, as warnings are OOS for this issue